lfucg-exactions/client/js/components/AccountLedgerForm.js

```diff
@@ -48,6 +48,8 @@ class AccountLedgerForm extends React.Component {
             accountFormChange,
         } = this.props;
 
+        const currentParam = this.props.params.id;
```

Comment:
It's not clear (to me at least) what this is being used for.  I see it is being used on line 113 below, but is it needed?  What does it represent?

Reply:
This is checking that the page has an id parameter in the url.  This indicates that it is an existing entry for updating vs. a new entry.  It prevents the display of the "Approval Pending" warning.